src/components/SelectableVisualization.vue

```diff
@@ -259,9 +270,44 @@ export default {
       }
       return date;
     },
+    timeline_daterange: function () {
+      let date = this.activityStore.query_options.date;
+      if (!date) {
+        date = this.activityStore.query_options.timeperiod.start;
+      }
+
+      // Get events data from CurrentData ± 1 Day so that there is some continuity
+      return [moment(date).subtract(1, 'day'), moment(date).add(1, 'day')];
+    },
     isSingleDay: function () {
       return _.isEqual(this.activityStore.query_options.timeperiod.length, [1, 'day']);
     },
   },
+  watch: {
+    timeline_daterange: async function () {
```

Comment:
This is not in data or a computed? Dead code?

Reply:
if you mean timeline_daterange, its there as computed on line 273

Reply:
Ahh sorry, my bad, I thought it was a method.